apps/web/src/views/CakeStaking/components/BenefitCard.tsx

```diff
@@ -134,7 +137,7 @@ export const BenefitCard: React.FC<{
           <Text lineHeight="130%">
             <StyleUl>
               {info.desc.map((item) => (
-                <li key={item}>{t(item)}</li>
+                <li key={item?.toString()}>{item}</li>
```

Comment:
What's the result of `toString()` call on a `ReactNode`?

Reply:
`Object`😂
Fixed.